### PR TITLE
Add channel_updated websocket action

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Handle channel updated WebSocket events

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-7677

#### Checklist
- [x] Added or updated unit tests (required for all new features)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ check-style: | pre-run .npminstall
 clean:
 	@echo Cleaning app
 
-	npm cache clean
+	npm cache clean --force
```
Comment (jwilander) [inline Makefile]: What are the makefile changes needed for?

File: src/actions/websocket.js [hunk 2]
```diff
@@ -457,6 +460,28 @@ function handleChannelDeletedEvent(msg, dispatch, getState) {
...
+
+            if (currentChannelId === channel.id) {
+                // Emit an event with the channel received as we need to handle
+                // the changes without listening to the store
+                EventEmitter.emit(WebsocketEvents.CHANNEL_UPDATED, channel);
+            }
+        }
+    } catch (err) {
+        // do nothing
+    }
```
Comment (jwilander) [inline src/actions/websocket.js]: Is the try catch just for the JSON parsing? If so, maybe just put it around that and return in the catch

File: Makefile (hunk 1 above)
Comment (enahum) [reply Makefile]: npm 5 complains, that's why I added it

File: src/actions/websocket.js (hunk 2 above)
Comment (enahum) [reply src/actions/websocket.js]: will do

**Pull request merged**